fetchUser(userId)
	.then(user => {
		return processUser(user);
	})
	.catch(error => {
		console.error('Error:', error);
		throw error;
	})